sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_connection_async.py

```diff
@@ -236,7 +236,7 @@ async def _read_frame(self, wait: Union[bool, int, float] = True, **kwargs) -> b
         """
         timeout: Optional[Union[int, float]] = None
         if wait is False:
-            timeout = 1  # TODO: What should this default be?
+            timeout = .2  # TODO: What should this default be?, are the other if statements are used 
```

Comment:
What prompted this value?

Reply:
This was a change made to align the read timeout on the async side to the read timeout on the sync side that had previously been made before Event Hubs went GA. I was seeing receive statements come in at 1 second increments on the async side (significantly slower than uamqp) and after debugging with Swathi and Kashif we realized that sync and async didn't align.